style.css

```diff
@@ -879,4 +1015,117 @@ body {
     margin-top: 25px;
     color: #253858;
   }
+
+  /* contact me section */
+
+  .contact-me {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: -30px;
+    background-image: url("./images/contactMe-bg.svg");
+    background-repeat: no-repeat;
+    background-size: 68%;
+    background-position: right top 40%;
+    background-color: #6070ff;
+    min-height: 100vh;
+    margin-top: -80px;
+    border-radius: 115px 0 0 0;
+    padding-top: 100px;
+  }
+
+  .contact-title {
+    color: white;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    margin-top: 50px;
+  }
+
+  .contact-desc {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    color: #ebebff;
+    padding: 0 24px;
+    margin-bottom: 5px;
+  }
+
+  .form {
+    width: 100%;
+    margin: 0 24px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .form form {
+    display: flex;
+    flex-direction: column;
+    width: 85%;
+  }
+
+  .form-input {
+    height: 48px;
+    margin-bottom: 24px;
+    border-radius: 8px;
+    padding-left: 15px;
+    color: rgba(179, 186, 197, 1);
+  }
+
+  .form-input:focus {
+    border: 2px solid #396df2;
+    border-radius: 8px;
+    color: rgba(23, 43, 77, 1);
+    font-family: 'Poppins', sans-serif;
+    font-size: regular;
+    font-size: 17px;
+    line-height: 20px;
+    line-height: 78%;
+  }
+
+  #form-commnt {
+    height: 170px;
+    border-radius: 8px;
+    margin-bottom: 24px;
+    padding-left: 15px;
+    padding-top: 15px;
+    font-family: 'Poppins', sans-serif;
+    font-style: regular;
+    font-size: 15px;
+    line-height: 24px;
+    line-height: 107%;
+    color: #b3bac5;
+  }
+
+  .btn-submit {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    padding-left: 18px;
+    gap: 10px;
+    width: 150px;
+    height: 60px;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    text-align: center;
+    letter-spacing: 0.03em;
+    text-decoration: none;
+    color: #396df2;
+    margin-bottom: 50px;
+  }
```

Comment:
- [ ] Please 🙏 make sure button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are styled as shown in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=jMI1dS3fpo2Vhw0e-0). Your pressed state button interaction background should be as in the figma design currently you are using the same background color for both hover and active 

<img width="404" alt="Screenshot 2023-02-22 at 5 00 42 AM" src="https://user-images.githubusercontent.com/44798044/220486937-8b033473-7e75-4bed-89d7-f7b0dfb3e042.png">